### DISCUSSION TITLE
why sample-live-wallpaper lunch succeeded but my Android 13 phone doesn't display？

### DISCUSSION THREAD

**Discussion by JiangnanFu** [google/filament#6411] [General]

**Comment by romainguy** [comment]
How are you starting it?

**Reply by JiangnanFu** [reply]
just click the button of run; all samples but sample-live-sample can display on my android 13 phone, Android studio indicates lunch succeeded，but nothing change on phone

**Reply by romainguy** [reply]
You need to set the live wallpaper in the home screen

**Reply by JiangnanFu** [reply]
I set the live wallpaper successfully by use google "wallpaper" app, thanks！
